.github/workflows/release.yaml

```diff
@@ -14,15 +14,21 @@ jobs:
           go-version: 1.14
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Unshallow # This step is required for the changelog to work correctly
-        run: git fetch --prune --unshallow
+        with:
```

Comment:
This has changed in gorelease@v2 to properly generate changeslog from git commit messages